frontend/lib/src/components/core/Block/Block.tsx

```diff
@@ -75,7 +75,11 @@ const BlockNodeRenderer = (props: BlockPropsWithWidth): ReactElement => {
   )
 
   const childProps = { ...props, ...{ node } }
-  const child: ReactElement = <LayoutBlock {...childProps} />
+  // TODO: We will set the value to disallow fullscreen mode within dialogs in a follow-up PR
+  const hideFullScreenButton = false
```

Comment:
nit - possible refactoring: we could potentially move all `libConfig.disableFullscreenMode` uses to its location, making it the only location where the libconfig option is used:

```suggestion
  const hideFullScreenButton = libConfig.disableFullscreenMode or <is dialog>
```